stdlib/public/Platform/Windows/SwiftWinDbg/SwiftWinDbg.cpp

```diff
@@ -0,0 +1,109 @@
+#define WIN32_LEAN_AND_MEAN // Exclude rarely-used stuff from Windows headers
+#include <windows.h>
+#define KDEXT_64BIT
+#include <dbgeng.h>
+#include <wdbgexts.h>
+#include <stdint.h>
+
+WINDBG_EXTENSION_APIS ExtensionApis = {0};
+EXT_API_VERSION g_ExtApiVersion = {1, 0, EXT_API_VERSION_NUMBER64, 0};
+USHORT SavedMajorVersion, SavedMinorVersion;
+
+bool APIENTRY DllMain(HMODULE hModule, DWORD ul_reason_for_call,
+                      LPVOID lpReserved) {
+  switch (ul_reason_for_call) {
+  case DLL_PROCESS_ATTACH:
+  case DLL_THREAD_ATTACH:
+  case DLL_THREAD_DETACH:
+  case DLL_PROCESS_DETACH:
+    break;
+  }
+  return true;
+}
+
+LPEXT_API_VERSION WDBGAPI ExtensionApiVersion(void) { return &g_ExtApiVersion; }
+
+void WDBGAPI WinDbgExtensionDllInit(PWINDBG_EXTENSION_APIS64 lpExtensionApis,
+                                    USHORT MajorVersion, USHORT MinorVersion) {
+  ExtensionApis = *lpExtensionApis;
+  SavedMajorVersion = MajorVersion;
+  SavedMinorVersion = MinorVersion;
```

Comment:
What are `SavedMajorVersion` and `SavedMinorVersion` used for?

Reply:
I don't think they are needed. I put them in because I found them in other basic WinDbg extension code.